frontend/src/routes/[[lang=locale]]/candidate/password-reset/PasswordResetPage.svelte

```diff
@@ -67,34 +63,15 @@
     <h1 class="text-xl font-normal">{$page.data.election.name}</h1>
     <h1 class="my-24 text-2xl font-normal">{$t('candidateApp.resetPassword.createNewPassword')}</h1>
   </HeadingGroup>
-
-  <form class="flex-nowarp flex flex-col items-center" on:submit|preventDefault={onButtonPress}>
-    <p class="m-0 max-w-md text-center">
-      {$t('candidateApp.setPassword.description')}
-    </p>
-
-    <PasswordValidator bind:validPassword {password} />
-
-    <label for="password" class="hidden">{$t('candidate.password')}</label>
-    <label for="passwordConfirmation" class="hidden">{$t('candidate.password')}</label>
-    <div class="mb-md flex w-full max-w-md flex-col gap-6">
-      <PasswordField bind:password autocomplete="new-password" />
-      <PasswordField bind:password={passwordConfirmation} autocomplete="new-password" />
-    </div>
-    {#if errorMessage}
-      <p class="text-center text-error">
-        {errorMessage}
-      </p>
-    {/if}
-
-    <Button
-      type="submit"
-      variant="main"
-      disabled={disableSetButton}
-      text={$t('candidateApp.setPassword.setPassword')} />
-
-    <Button href={$getRoute(Route.CandAppHelp)} text={$t('candidate.contact_support')} />
+  <form class="flex-nowarp flex flex-col items-center">
+    <PasswordSetComponent
+      on:ButtonPressed={onButtonPress}
+      bind:validPassword
+      bind:errorMessage
+      bind:password1={password}
+      bind:password2={passwordConfirmation}
+      autocomplete1="new-password"
+      autocomplete2="new-password" />
   </form>
-
   <svelte:fragment slot="footer"><Footer /></svelte:fragment>
```

Comment:
Not part of the diff but could this be simplified to just ` <Footer slot="footer" />`